mods/d2k/weapons.yaml

```diff
@@ -455,24 +455,21 @@ DeviatorMissile:
 	ReloadDelay: 160
 	Range: 5c0
 	Report: MISSLE1.WAV
-	InvalidTargets: Infantry, Structure
 	Projectile: Missile
 		MaximumLaunchSpeed: 281
-		RangeLimit: 40
```

Comment:
Yeah the Missile Tank doesnt have it which is why I removed it. Quads also miss ridiculously often but that's another issue.